### PR TITLE
feat: add Internationalization (i18n)

### PR DESCRIPTION
**Pull request by Aliaksandr-Shliayeu**
### Internationalization (i18n)

**Added**:

1. English locale

2.  Russian locale

### Pages

**Covered**:
Fully:
- NavBar 
- SignIn/SignUp 

Partially:
- other pages

### CODE REVIEW

File: js_machine_front/src/components/authorization/SignInFrom.tsx [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
-            <div className="auth-form__title">или используйте E-mail для регистрации:</div>
-            <input className="auth-form__input" type="text" placeholder="E-mail" />
-            <input className="auth-form__input" type="password" placeholder="Пароль" />
-            <button className="auth-form__btn" type="submit">ВОЙТИ</button>
-        </form>
+            <div className="auth-form__title">
+                <FormattedMessage id="authorization.signInFormTitle" />
+            </div>
+            <FormattedMessage id="authorization.signInFormEmail" >
+                { placeholder => <input className="auth-form__input" type="email" placeholder={placeholder as string}/>}
```
Comment (vharadkou) [inline js_machine_front/src/components/authorization/SignInFrom.tsx]: better to use (placeholder: string) =>

File: js_machine_front/src/i18n/messages.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+export const messages: {[index: string]: any} = {
```
Comment (vharadkou) [inline js_machine_front/src/i18n/messages.ts]: no any, you can cretae interface {['en-Us']: {[index: string]: string}}

**Review by vharadkou — CHANGES_REQUESTED** [review]

**Review by vharadkou — APPROVED** [review]

File: js_machine_front/src/i18n/messages.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Aliaksandr-Shliayeu) [reply js_machine_front/src/i18n/messages.ts]: fixed

File: js_machine_front/src/components/authorization/SignInFrom.tsx (hunk 1 above)
Comment (Aliaksandr-Shliayeu) [reply js_machine_front/src/components/authorization/SignInFrom.tsx]: fixed

File: js_machine_front/src/i18n/messages.ts (hunk 2 above)
Comment (Aliaksandr-Shliayeu) [reply js_machine_front/src/i18n/messages.ts]: fixed

**Pull request merged**